### DISCUSSION TITLE
%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%989/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#12] [Comments]
# %EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%989/

양성빈의 블로그

https://sungbin.kr/%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%989/

**Comment by tdronnie** [comment]
메서드 오버로딩을 잘 사용하면 코드의 중복은 줄이면서 더 짜임새있는 로직을 작성할 수 있을 것 같네요! 정리 잘 보았습니다 :)

**Reply by SungbinYang** [reply]
감사합니다!
